k8s/templates/hos-proxy.yaml

```diff
@@ -58,6 +58,20 @@ spec:
           - mountPath: /run/spire/sockets
             name: spire-agent-socket
             readOnly: true
+        - name: grpcurl
+          command:
```

Comment:
Not sure if there's a `noop` command, but I was picturing using a `readinessProbe` instead - something more like:

```yaml
  name: connection-check
  readinessProbe:
    exec:
      command:
        - grpcurl
        - "-plaintext"
        - localhost:5000
        - list
      periodSeconds: 15
```

That way we'll still get pings on our side, but the client will also be able to see if the pod is healthy on their side.

Reply:
maybe this would work for the `command`: https://stackoverflow.com/a/35770783/817950

```yaml
command:
  - exec
  - /bin/sh
  - "-c"
  - "trap : TERM INT; sleep infinity & wait"
```

Reply:
Would it make sense to do both, so that you can both see the logs and they would have visibility into whether there's an error or not?

Maybe to avoid duplicate pings the script can output to `/status` and the readinessProbe can just `grep -v Error /status` or something